IPublicable.php

```diff
@@ -0,0 +1,4 @@
+<?php
+interface IPublicable {
```

Comment:
```suggestion

interface IPublicable {
```
Para una mejor interfaz de código dejar un espacio al iniciar php